app/services/foreman_templates/uninstaller.rb

```diff
@@ -0,0 +1,13 @@
+module ForemanTempaltes
+  class Cleaner
```

Comment:
Cleaner != "uninstaller.rb" so it isn't autoloaded. Renaming the file to "cleaner.rb" works